api/post_test.go

```diff
@@ -138,6 +138,37 @@ func TestCreatePost(t *testing.T) {
 	}
 }
 
+func TestCreatePostWithCreateAt(t *testing.T) {
+
+	// An ordinary user cannot use CreateAt
+
+	th := Setup().InitBasic()
```

Comment:
Looks good, but I have a small change just to simplify things. If you change this to `th := Setup().InitBasic().InitSystemAdmin()`, you can use the same `TestHelper` below for `SysClient := th.SystemAdminClient`